Marlin/src/module/tool_change.cpp

```diff
@@ -424,6 +416,13 @@ void tool_change(const uint8_t tmp_extruder, const float fr_mm_s/*=0.0*/, bool n
             }
           #endif
 
+          #if ENABLED(SWITCHING_NOZZLE)
+            // Always raise by at least 1 to avoid workpiece
+            current_position[Z_AXIS] += ( (zdiff > 0.0 ? 0.0 : -zdiff) + settings.switching_nozzle_raise_hotend );
```

Comment:
No such thing as `settings.switching_nozzle_raise_hotend` in Marlin at this time.